pkg/generator/generator.go

```diff
@@ -186,3 +244,68 @@ func SelectFromCommand() ([]string, error) {
 
 	return targets, nil
 }
+
+// checkAvailableRegions checks if regions in target configuration exist in regions configuration
+func CheckAvailableRegions(regions []string, targets []schema.Target) error {
+	for _, target := range targets {
+		if target.Regions != nil && len(target.Regions) > 0 {
+			for _, region := range target.Regions {
```

Comment:
If it possible, how think about change sub loop to function?

Reply:
I think it is a good idea. 
But can you please let me know the reason why it's better? I would like to know the reason in order to make function correctly.

Reply:
Maybe, it would be work without any problem.
But, I think If you create function for that, it will be better for reading. (for human)

ex) func hasUnsupportedRegion(target schema.Target) boolean { }

But it's just my personal opinion. I don't know what is the best choice for good code. :)

Reply:
Thanks for your explanation! It would be better to change as you mentioned.
I will update it!

Reply:
Thank you for applying.
Sometimes, some code change has no effect to result.
But I think it's very fun time that thinking about good(?) code, even I don't know exactly what is the best choice or best practice.